README.md

```diff
@@ -47,6 +47,24 @@ To create a package from scratch, refer to the nixpkgs documentation available a
 
 ## Bundles and jobs
 
+Bundles are defined in your `config.nix` file. If you didn't change this part of the auto-generated file, you have a single bundle called "default".
+Bundles are used to create sets of compatible packages. You can override the version of some packages and you can explicitly exclude some incompatible packages.
+
+Jobs represent buildable outputs. You can build any package in `coqPackages` (including any package defined in your `.nix/coq-overlays` directory) with the following command:
+
+```
+nix-build --argstr job PACKAGENAME
+```
+
+If the package depends on your main package, then it will use its local version as a dependency.
+
+You can also specify the bundle to use like this:
+```
+nix-build --argstr bundle BUNDLENAME --argstr job PACKAGENAME
+```
+
+A default bundle is defined in `config.nix` and is used if you do not specify it on the command-line.
```

Comment:
It is used only if you do `nix-build` without arguments, otherwise if you indicate a job, it will build it for every builder. Do you think I should change this behavior and specialize to the current builder?

Reply:
Defining a default bundle becomes of limited interest in that case. What about having a special (virtual) bundle "all" that you can use if you really want to build a job for every bundle? That being said, I don't have a strong opinion here because I don't have any experience yet working with multiple bundles.

Reply:
I could do have a virtual `_all` bundle indeed...

Reply:
Let me know if you want me to fix the doc to reflect the current state, or if you want to tweak the behavior instead.

Reply:
fixed the code in #27